articles/20241126_Using_Environmental_Variables_in_Daytona.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Using Environmental Variables in Daytona'
+description: 'Learn how to set, manage, and use environmental variables in Daytona workspaces with a simple Python project for demonstration.'
+date: 2024-11-26
+author: 'Busayo Samuel'
+tags: ['Environment Variables', 'Development Environment', 'Daytona Workspaces']
+---
+
+# Using Environmental Variables in Daytona
+
+## Introduction
+
+[Environmental variables]('/definitions/20241126_definition_environment_variables.md') are important for configuring development environments and storing sensitive information. They provide a flexible way to control application behavior without hardcoding values directly into the source code. Daytona offers a streamlined method for setting, managing, and using environmental variables across various workspaces using the `daytona env` command. This simplifies configuration and enhances security especially when collaborating on projects.
```

Comment:
please move the link from the first paragraph somewhere else, especially when it is a first word it will look ugly, thanks @bellatrick